config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -438,28 +449,85 @@ periodics:
         org:kubernetes-csi
+        is:issue
         -repo:kubernetes-sigs/kind
         -repo:kubernetes/ingress-nginx
         -label:lifecycle/frozen
         -label:lifecycle/stale
         -label:lifecycle/rotten
         -label:"help wanted"
         -label:"good first issue"
+        -label:"triage/accepted"
       - --updated=2160h
       - --token=/etc/github-token/token
       - --endpoint=http://ghproxy.default.svc.cluster.local
       - |-
-        --comment=The Kubernetes project currently lacks enough contributors to adequately respond to all issues and PRs.
+        --comment=The Kubernetes project currently lacks enough contributors to adequately respond to all issues.
 
-        This bot triages issues and PRs according to the following rules:
+        This bot triages un-triaged issues according to the following rules:
         - After 90d of inactivity, `lifecycle/stale` is applied
         - After 30d of inactivity since `lifecycle/stale` was applied, `lifecycle/rotten` is applied
         - After 30d of inactivity since `lifecycle/rotten` was applied, the issue is closed
 
         You can:
-        - Mark this issue or PR as fresh with `/remove-lifecycle stale`
-        - Mark this issue or PR as rotten with `/lifecycle rotten`
-        - Close this issue or PR with `/close`
+        - Mark this issue as fresh with `/remove-lifecycle stale`
+        - Close this issue with `/close`
+        - Offer to help out with [Issue Triage][1]
+
+        Please send feedback to sig-contributor-experience at [kubernetes/community](https://github.com/kubernetes/community).
+
+        /lifecycle stale
+
+        [1]: https://www.kubernetes.dev/docs/guide/issue-triage/
+      - --template
+      - --ceiling=10
+      - --confirm
+      volumeMounts:
+      - name: token
+        mountPath: /etc/github-token
+    volumes:
+    - name: token
+      secret:
+        secretName: k8s-triage-robot-github-token
+
+- name: ci-k8s-triage-robot-stale-prs
+  interval: 1h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  annotations:
+    testgrid-dashboards: sig-contribex-k8s-triage-robot
+    description: Adds lifecycle/stale to PRs after 90d of inactivity
+    testgrid-tab-name: stale-prs
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/commenter:v20230207-1f583294de
+      command:
+      - commenter
+      args:
+      - |-
+        --query=org:kubernetes
+        org:kubernetes-sigs
+        org:kubernetes-client
+        org:kubernetes-csi
+        -repo:kubernetes-sigs/kind
+        -repo:kubernetes/ingress-nginx
+        -label:lifecycle/frozen
+        -label:lifecycle/stale
+        -label:lifecycle/rotten
+      - --updated=2160h
+      - --token=/etc/github-token/token
+      - --endpoint=http://ghproxy.default.svc.cluster.local
+      - |-
+        --comment=The Kubernetes project currently lacks enough contributors to adequately respond to all PRs.
+
+        This bot triages PRs according to the following rules:
```

Comment:
@tallclair I think this is missing is:pr

Reply:
Xref https://github.com/kubernetes-sigs/cluster-api/issues/5863#issuecomment-1422034944